akka-cluster/src/multi-jvm/scala/akka/cluster/routing/ClusterConsistentHashingGroupSpec.scala

```diff
@@ -16,8 +16,8 @@ import scala.concurrent.Await
 
 object ClusterConsistentHashingGroupMultiJvmSpec extends MultiNodeConfig {
 
-  case object Get
-  final case class Collected(messages: Set[Any])
+  case object Get extends JavaSerializable
```

Comment:
Any reason for the ones we use Cbor vs java in tests for?

Reply:
added comment: using Java serialization because of `Any` in `Collected` (don't want to spend time on rewriting test)

`Get` would work with Cbor but thought it was best to use same